.travis.yml

```diff
@@ -74,6 +74,14 @@ before_script:
   - TEST_TIMEZONES=("America/New_York" "Asia/Calcutta" "UTC")
   - TEST_TIMEZONE=${TEST_TIMEZONES["`shuf -i 0-2 -n 1`"]}
 
+install:
+    # Because of either some changes in travis, composer or git, composer is not able to pick version for "self" on inclusion of solr anymore, so we force it:
+  -  if [ "$TEST_CONFIG" != "" ] ; then export COMPOSER_ROOT_VERSION=`php -r 'echo json_decode(file_get_contents("./composer.json"), true)["extra"]["branch-alias"]["dev-tmp_ci_branch"];'` ; fi
```

Comment:
Did you try without this?
Afaik this has since been resolved, so cool if you can try removing those two lines in a commit and see if all jobs still pass.